deployments/arbitrum/usdc/relations.ts

```diff
@@ -0,0 +1,20 @@
+import { RelationConfigMap } from '../../../plugins/deployment_manager/RelationConfig';
+import baseRelationConfig from '../../relations';
+
+export default {
+  ...baseRelationConfig,
+  'governor': {
+    artifact: 'contracts/bridges/arbitrum/ArbitrumBridgeReceiver.sol:ArbitrumBridgeReceiver',
+  },
+  ClonableBeaconProxy: {
+    artifact: 'contracts/ERC20.sol:ERC20',
+  },
+  TransparentUpgradeableProxy: {
+    artifact: 'contracts/ERC20.sol:ERC20',
```

Comment:
This relation should probably be specific to a single alias, since not all `TransparentUpgradeableProxy` contracts are ERC20s

Reply:
Yeah probably better to hardcode the address of the contract you are defining the relations for here. e.g. `0xabc...` instead of `TransparentUpgradeableProxy`

We do something similar for Optimism testnet.

Reply:
Do you know which addresses we are setting this relation for?

Reply:
i don't remember, no